package.json

```diff
@@ -22,6 +22,7 @@
     "cross-env": "^5.1.5",
     "css-loader": "^0.28.11",
     "del": "^3.0.0",
+    "dom-scroll-into-view": "^1.2.1",
```

Comment:
Isn't this already packaged with WordPress?

Reply:
Good question. I couldn't find evidence that it's packed with WordPress, though maybe it is.

For example, the `@wordpress/components` [package](https://github.com/WordPress/gutenberg/blob/3397ae408a870ded1516fcea77b63cdafb72f873/packages/components/package.json) includes from an external source, not WordPress:

```js
"dom-scroll-into-view": "^1.2.1",
```